commands/pipelines/setup.js

```diff
@@ -159,6 +159,66 @@ function* getCISettings (yes, organization) {
   return settings
 }
 
+function createApps (heroku, archiveURL, pipeline, pipelineName, stagingAppName, organization) {
+  const prodAppSetupPromise = createApp(heroku, {
+    archiveURL,
+    pipeline,
+    name: pipelineName,
+    stage: 'production',
+    organization
+  })
+
+  const stagingAppSetupPromise = createApp(heroku, {
+    archiveURL,
+    pipeline,
+    name: stagingAppName,
+    stage: 'staging',
+    organization
+  })
+
+  const promises = [prodAppSetupPromise, stagingAppSetupPromise]
+
+  return Promise.all(promises).then(appSetups => {
+    return appSetups
+  }, (error) => {
+    cli.exit(1, error)
+  })
+}
+
+function wait(ms) {
+  return new Promise((resolve, reject) => setTimeout(resolve, ms))
+}
+
+function pollAppStatus (heroku, build) {
```

Comment:
We keep using `build` but I think the resource here is an `appSetup`. We should consider changing the name to avoid confusing the next person.